Tests/Request/ParamFetcherTest.php

```diff
@@ -406,7 +406,7 @@ public function testParamContainerDefinition()
         $param2->expects($this->once())
             ->method('getName')
             ->willReturn('bar');
-        $param2->expects($this->once())
+        $param2->expects($this->any())
```

Comment:
not really sure how PHPUnit deals with traits at this point .. maybe better just remove this ``expects()`` entirely now?